### PR TITLE
Update CaseList and CarbonTable

### PR DESCRIPTION
**Pull request by teodor-ritense**

### CODE REVIEW

**Comment by mbritense** [comment]
Column titles are not translated

**Comment by mbritense** [comment]
Query params / pagination / assignee filters are not recovered from URL

**Comment by mbritense** [comment]
Arrows don't clear when sorting on another column

**Comment by teodor-ritense** [comment]
> Column titles are not translated

Updated

**Comment by teodor-ritense** [comment]
> Query params / pagination / assignee filters are not recovered from URL

Updated

**Comment by teodor-ritense** [comment]
> Arrows don't clear when sorting on another column

Updated

**Review by Klaas-Ritense — APPROVED** [review]

File: projects/valtimo/plugin/src/lib/plugins/verzoek/components/verzoek-configuration/verzoek-configuration.component.html [hunk 1]
```diff
@@ -40,12 +40,26 @@
...
+      [title]="'notificatiesApiPluginConfiguration' | pluginTranslate: pluginId | async"
       [disabled]="obs.disabled"
       [defaultSelectionId]="obs.prefill?.notificatiesApiPluginConfiguration"
       [required]="true"
       [loading]="!vars.notificatiePluginSelectItems"
-      [tooltip]="'notificatiesApiPluginConfigurationTooltip' | pluginTranslate : pluginId | async"
+      [tooltip]="'notificatiesApiPluginConfigurationTooltip' | pluginTranslate: pluginId | async"
+    ></v-select>
+  </ng-container>
+  <ng-container *ngIf="{objectManagementSelectItems: objectManagementSelectItems$ | async} as vars">
```
Comment (Klaas-Ritense) [inline projects/valtimo/plugin/src/lib/plugins/verzoek/components/verzoek-configuration/verzoek-configuration.component.html]: This has moved to line 144

**Review by Klaas-Ritense — CHANGES_REQUESTED** [review]
objectManagement has moved

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**